wiki/People/Centurions/fr.md

```diff
@@ -25,18 +23,18 @@ Les Centurions sont des utilisateurs qui ont atteint la prestigieuse étape de c
-| ::{ flag=US }:: [Nevo](https://osu.ppy.sh/users/7451883) | [09/12/2017](https://osu.ppy.sh/beatmapsets/623924) | [24/11/2020](https://osu.ppy.sh/beatmapsets/1205168) | [#1](https://osu.ppy.sh/beatmaps/packs/TM3)  |
-| ::{ flag=CN }:: [Ryuusei Aika](https://osu.ppy.sh/users/7777875) | [10/06/2017](https://osu.ppy.sh/beatmapsets/598791) | [02/10/2022](https://osu.ppy.sh/beatmapsets/1802635) | [#1](https://osu.ppy.sh/beatmaps/packs/TM4)  |
-| ::{ flag=RU }:: [SMOKELIND](https://osu.ppy.sh/users/9327302) | [11/12/2018](https://osu.ppy.sh/beatmapsets/879904) | [20/03/2023](https://osu.ppy.sh/beatmapsets/1916502) | [#1](https://osu.ppy.sh/beatmaps/packs/TM2)  |
+| ::{ flag=US }:: [Nevo](https://osu.ppy.sh/users/7451883) | [09/12/2017](https://osu.ppy.sh/beatmapsets/623924) | [24/11/2020](https://osu.ppy.sh/beatmapsets/1205168) | [#1](https://osu.ppy.sh/beatmaps/packs/TM3) |
+| ::{ flag=CN }:: [Ryuusei Aika](https://osu.ppy.sh/users/7777875) | [10/06/2017](https://osu.ppy.sh/beatmapsets/598791) | [02/10/2022](https://osu.ppy.sh/beatmapsets/1802635) | [#1](https://osu.ppy.sh/beatmaps/packs/TM4) |
+| ::{ flag=RU }:: [SMOKELIND](https://osu.ppy.sh/users/9327302) | [11/12/2018](https://osu.ppy.sh/beatmapsets/879904) | [20/03/2023](https://osu.ppy.sh/beatmapsets/1916502) | [#1](https://osu.ppy.sh/beatmaps/packs/TM2) |
 | ::{ flag=US }:: [Secre](https://osu.ppy.sh/users/2306637) | [16/09/2018](https://osu.ppy.sh/beatmapsets/826551) | [30/12/2022](https://osu.ppy.sh/beatmapsets/1348272) |  |
 | ::{ flag=FR }:: [Sotarks](https://osu.ppy.sh/users/4452992) | [13/12/2015](https://osu.ppy.sh/beatmapsets/373858) | [29/08/2018](https://osu.ppy.sh/beatmapsets/809788) | [#1](https://osu.ppy.sh/beatmaps/packs/TM5), [#2](https://osu.ppy.sh/beatmaps/packs/TM6), [#3](https://osu.ppy.sh/beatmaps/packs/TM7) |
-| ::{ flag=US }:: [wafer](https://osu.ppy.sh/users/9416836) | [12/07/2019](https://osu.ppy.sh/beatmapsets/965596) | [07/10/2023](https://osu.ppy.sh/beatmapsets/2023741) | [#1](https://osu.ppy.sh/beatmaps/packs/TM18)  |
+| ::{ flag=US }:: [wafer](https://osu.ppy.sh/users/9416836) | [12/07/2019](https://osu.ppy.sh/beatmapsets/965596) | [07/10/2023](https://osu.ppy.sh/beatmapsets/2023741) | [#1](https://osu.ppy.sh/beatmaps/packs/TM18) |
 | ::{ flag=US }:: [ztrot](https://osu.ppy.sh/users/6347) | [09/03/2009](https://osu.ppy.sh/beatmapsets/5417) | [08/03/2012](https://osu.ppy.sh/beatmapsets/44666) |  |
 
 ## Le saviez-vous ?
 
 - Le premier utilisateur à avoir atteint 100 beatmaps classés est [DJPop](https://osu.ppy.sh/users/2363), le 4 août 2010.
-- Parmi tous les Centurions, il faut environ **3 ans et demi** (environ 1306 jours) pour classer 100 beatmaps en moyenne.
+- Parmi tous les Centurions, il faut environ **3 ans et demi**<!-- "x,x ans" or "x ans et demi" (6 months) --> (environ 1306 jours) pour classer 100 beatmaps en moyenne.
```

Comment:
```suggestion
- Parmi tous les Centurions, il faut environ **3,6 ans **<!-- "x,x ans" or "x ans et demi" (6 months) --> (environ 1306 jours) pour classer 100 beatmaps en moyenne.
```

disagree with the usage of `demi` here because it means "half", which isn't the case here because that only works if the number is 3.5